package.json

```diff
@@ -42,8 +44,7 @@
     "button"
   ],
   "dependencies": {
-    "ember-cli-babel": "^5.1.6",
-    "ember-cli-htmlbars": "1.0.3"
```

Comment:
looks like I moved it in another direction as part of the ember-cli 2.4 upgrade https://github.com/yapplabs/ember-radio-button/commit/085e4aa30d6ba7286973f357e0743d83bded0a98